src/util/config.tsx

```diff
@@ -0,0 +1,101 @@
+import { LxcConfigOptionCategories, LxdConfigField } from "types/config";
+
+export const toConfigFields = (
+  categories: LxcConfigOptionCategories,
+): LxdConfigField[] => {
+  return Object.values(categories).flatMap((value) =>
```

Comment:
just a personal preference so not critical to change, I think the function might be easier to read if we use normal for loops. Having the tests helps with understanding though

Reply:
True, this used to be hard to read before. I'll change it.